app/views/dashboard/index.html.erb

```diff
@@ -1,13 +1,8 @@
-<% if(logged_in?) %>
-  <%= react_component('Dashboard', {user: current_user,
-    projects: @projects,
-    portfolio: current_user.portfolio,
-    avatar: @avatar,
-    project_avatars: @project_avatars,
-    markdownInfo: @markdown_info},
-  {prerender: true}) %>
-<% else %>
-  <%= react_component('Main', {user: current_user,
-    signed_in: 0},
-  {prerender: true}) %>
-<% end %>
+<%= react_component('Dashboard', {
+  user: current_user,
+  projects: @dashboard_presenter.projects,
+  portfolio: current_user.portfolio,
```

Comment:
@drod1000 I think we could utilize here the PORO as well, you already wrote the method. So make this line: `portfolio: @dashboard_presenter.portfolio,`


FYI @mollybrown @jdconrad89